### DISCUSSION TITLE
Safari animation issue

### DISCUSSION THREAD

**Discussion by woodrunsdeep** [nolimits4web/swiper#6566] [Q&A]
Card doesn't scale smoothly as can be seen on video below and [here](https://woodrunsdeep-kartuli-anbani.netlify.app/) live on a page. 
On video I slowed down slide change so scale issue can be seen clearly. 
Source code can be found [here](https://github.com/woodrunsdeep/kartuli-anbani).

I scale slides like this:
```
.swiper-slide {
  transition: transform 450ms ease-in-out;
}

.swiper-slide-active {
  transform: scale(1.1);
}
```

I encounter this issue only with iOS. Chrome, Edge and Safari on macOS handle animation just fine.

It doesn't look like Swiper's issue but since I'm using it I thought maybe someone here could help me out. 
It's quite possible that my implementation is not done correctly and I don't see my mistake.
Have anyone encounter such behavior? Any feedback will be much appreciated 🙏

Swiper: 8.4.7
iOS: 16.3 (previous versions also behave this way)

https://user-images.githubusercontent.com/50412253/230895902-f6c989ba-fb1a-4c1d-84fc-41d7e7320094.mov

**Comment by ramsaysewell** [comment]
Hi @woodrunsdeep,

I would try copying the transition style over to the `.swiper-slide-active` to get this:
```
.swiper-slide-active {
   transform: scale(1.1);
   transition: transform 450ms ease-in-out;
}
```
Also, make sure that you're setting a default transform property on `.swiper-slide` as the transition needs a start and end value to animate between. 

If either of those solutions don't help, showing some more of your code would be sure to help us help you.
